### PR TITLE
Create index.html

### PR DESCRIPTION
**Pull request by MiguelTolino**
This is a test using Github

### CODE REVIEW

**Comment by MiguelTolino** [comment]
### **_I'm testing Github, thanks for let me fork_**

- [ ] OKKKK

- MarkDown es la polla @polla #

**Comment by MiguelTolino** [comment]
Let me know if you liked!!! :))))))

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# HASHCODE
```
Comment (MiguelTolino) [inline README.md:1]: This is a comment of a line

**Review by MiguelTolino — COMMENTED** [review]
### Header

File: README.md (hunk 1 above)
Comment (JavierCervilla) [reply README.md:1]: eres una perra

**Pull request closed without merging**